### ISSUE TITLE
[BUG] Hover Styling Missing with Chakra >= 2.4.0

### ISSUE THREAD

**Issue by justinforlenza** [csandman/chakra-react-select#219]
### Description

When using chakra 2.4.0 or greater the hover styling for options is no longer visible in chakra-react-select

Chakra 2.4.1
![image](https://user-images.githubusercontent.com/11709872/202860188-29624deb-0a27-4e56-b867-5f04f7819778.png)

Chakra 2.3.7
![image](https://user-images.githubusercontent.com/11709872/202860236-90c69fc0-fa15-4a50-98cf-bd228c64d7ca.png)

### chakra-react-select Version

4.4.0

### Link to Reproduction

https://codesandbox.io/s/chakra-react-select-missing-hover-style-584m52

### TypeScript?

- [X] Yes I use TypeScript

### Steps to reproduce

1. Install Charka UI >= 2.4.0 & Chakra React Select
2. Create select component
3. hover over select option

### Operating System

- [X] macOS
- [X] Windows
- [X] Linux
- [ ] iOS/iPadOS
- [ ] Android

**Comment by kicks321** [comment]
@csandman, so we just started integrating this library into our frontend project.

We have two applications, both running 4.4.0.  What's weird is one of them, hover styles works perfect.  In the other, it doesn't at all.

**Comment by kicks321** [comment]
@justinforlenza and @csandman, I was able to solve this problem using just a custom Chakra Style

```
    const chakraStyles: ChakraStylesConfig = {
        dropdownIndicator: (provided, state) => ({
            ...provided,
            background: state.isFocused ? 'blue.100' : provided.background,
            p: 0,
            w: '40px',
        }),
        placeholder: (provided, state) => ({
            ...provided,
            color: 'gray.500',
        }),
        option: (provided, state) => ({
            ...provided,
            _hover: {
                color: 'blue.500',
                bg: 'gray.100',
            },
        }),
    };
```

**Comment by csandman** [comment]
Sorry for the late response on this, I've been on vacation for the past week.  I published a new version ([`v4.4.1`](https://github.com/csandman/chakra-react-select/releases/tag/v4.4.1)) with the fix from @tzii which appears to do the trick, but if there are any further issues feel free to reach out!
